packages/gatsby-plugin-theme-ui/utils/preset-dictionary.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
```

Comment:
Note sure what is happening with the prettier hook to change the format of the keys.